_search-plugins/knn/knn-index.md

```diff
@@ -53,42 +53,44 @@ However, if you intend to just use painless scripting or a k-NN score script, yo
-(the native library) to use, and a map of parameters.
+(the library) to use, and a map of parameters.
 
 Mapping Parameter | Required | Default | Updatable | Description
 :--- | :--- | :--- | :--- | :---
 `name` | true | n/a | false | The identifier for the nearest neighbor method.
-`space_type` | false | "l2" | false | The vector space used to calculate the distance between vectors.
-`engine` | false | "nmslib" | false | The approximate k-NN library to use for indexing and search. Either "faiss" or "nmslib".
+`space_type` | false | l2 | false | The vector space used to calculate the distance between vectors.
+`engine` | false | nmslib | false | The approximate k-NN library to use for indexing and search. The available libraries are faiss, nmslib, and Lucene.
 `parameters` | false | null | false | The parameters used for the nearest neighbor method.
 
 ### Supported nmslib methods
 
 Method Name | Requires Training? | Supported Spaces | Description
 :--- | :--- | :--- | :---
-`hnsw` | false | "l2", "innerproduct", "cosinesimil", "l1", "linf" | Hierarchical proximity graph approach to Approximate k-NN search. For more details on the algorithm, [checkout this paper](https://arxiv.org/abs/1603.09320)!
+`hnsw` | false | l2, innerproduct, cosinesimil, l1, linf | Hierarchical proximity graph approach to Approximate k-NN search. For more details on the algorithm, see this [abstract](https://arxiv.org/abs/1603.09320).
 
 #### HNSW Parameters
 
-Paramater Name | Required | Default | Updatable | Description
+Parameter Name | Required | Default | Updatable | Description
 :--- | :--- | :--- | :--- | :---
 `ef_construction` | false | 512 | false | The size of the dynamic list used during k-NN graph creation. Higher values lead to a more accurate graph, but slower indexing speed.
 `m` | false | 16 | false | The number of bidirectional links that the plugin creates for each new element. Increasing and decreasing this value can have a large impact on memory consumption. Keep this value between 2-100.
 
-**Note** --- For *nmslib*, *ef_search* is set in the [index settings](#index-settings).
+For *nmslib*, *ef_search* is set in the [index settings](#index-settings).
+{: .note}
 
 ### Supported faiss methods
 
 Method Name | Requires Training? | Supported Spaces | Description
 :--- | :--- | :--- | :---
-`hnsw` | false | "l2", "innerproduct"* | Hierarchical proximity graph approach to Approximate k-NN search.
-`ivf` | true | "l2", "innerproduct" | Bucketing approach where vectors are assigned different buckets based on clustering and, during search, only a subset of the buckets are searched.
+`hnsw` | false | l2, innerproduct | Hierarchical proximity graph approach to Approximate k-NN search.
+`ivf` | true | l2, innerproduct | Bucketing approach where vectors are assigned different buckets based on clustering and, during search, only a subset of the buckets are searched.
```

Comment:
Change "only a subset of the buckets are searched" to "only a subset of the buckets is searched."